integration/presto/src/main/java/org/apache/carbondata/presto/PrestoCarbonVectorizedRecordReader.java

```diff
@@ -176,8 +191,16 @@ private void initBatch() {
     for (int i = 0; i < queryDimension.size(); i++) {
       ProjectionDimension dim = queryDimension.get(i);
       if (dim.getDimension().isComplex()) {
+        List<CarbonDimension> childDimensions =
+                dim.getDimension().getListOfChildDimensions();
+        ArrayList<StructField> childFields = new ArrayList<StructField>();
+        for (int ind = 0; ind < childDimensions.size(); ind++) {
+          ColumnSchema childSchema = childDimensions.get(ind).getColumnSchema();
```

Comment:
go to each file where new code is added and look for unused variable/function added (IDE can show in grey) and remove it.

Here childSchema is not used.

Reply:
done